### PR TITLE
Feat/update share by link buttons

### PR DESCRIPTION
**Pull request by zatteo**

### CODE REVIEW

File: packages/cozy-sharing/locales/en.json [hunk 1]
```diff
@@ -62,6 +62,13 @@
       "sync_to_mine": "Synchronise in my Cozy",
       "add_to_mine": "Add to my Cozy",
       "close": "Close"
+    },
+    "permissionLink": {
+      "title": "Allow anyone with the link to :",
```
Comment (Crash--) [inline packages/cozy-sharing/locales/en.json:68]: no space in english ;)

https://www.learnenglish.de/punctuation/punctuationtext.html#colon

File: packages/cozy-sharing/src/components/Recipient/Recipient.spec.jsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ describe('Recipient component', () => {
       onRevokeSelf
     })
 
-    fireEvent.click(getByText('Can Change'))
+    fireEvent.click(getByText('can change'))
```
[the remark this answers is not in the document]
Comment (Crash--) [reply packages/cozy-sharing/src/components/Recipient/Recipient.spec.jsx:73]: No https://github.com/cozy/cozy-libs/pull/1948/commits/2536713fe2ae73346b96b87147ae41fd08d9330d#diff-3941458839998fc16d259f454bd7096192e8f242652cf43eff2bb095e6c968a5R61 ^^'

**Review by Crash-- — APPROVED** [review]
Nice!

File: packages/cozy-sharing/src/components/Recipient/recipient.styl [hunk 3]
```diff
@@ -52,6 +52,15 @@
     .recipient-user
         font-size 1.125rem
 
+.avatar-link-icon-wrapper
```
[the remark this answers is not in the document]
Comment (zatteo) [reply packages/cozy-sharing/src/components/Recipient/recipient.styl]: Yes I can! There is everything in cozy-ui :o

**Pull request merged**